build-tools/packages/build-tools/src/common/tsCompile.ts

```diff
@@ -40,31 +70,60 @@ export function tsCompile({
 
 	let code = ts.ExitStatus.DiagnosticsPresent_OutputsSkipped;
 	if (commandLine && diagnostics.length === 0) {
+		// When specified overrides current directory's package.json type field so tsc may cleanly
```

Comment:
Nit (took me a couple of reads to parse correctly 😋)
```suggestion
		// When specified, overrides current directory's package.json type field so tsc may cleanly
```